static/app/components/modals/createNewIntegrationModal.tsx

```diff
@@ -107,19 +121,25 @@ function CreateNewIntegrationModal({
         <Button
           priority="primary"
           size="sm"
-          to={`/settings/${organization.slug}/developer-settings/${
-            option === 'public' ? 'new-public' : 'new-internal'
-          }/`}
+          to={
+            option === 'sentry-fx'
+              ? `/settings/${organization.slug}/developer-settings/sentry-functions/new/`
+              : `/settings/${organization.slug}/developer-settings/${
+                  option === 'public' ? 'new-public' : 'new-internal'
+                }/`
+          }
           onClick={() => {
-            trackIntegrationAnalytics(
-              option === 'public'
-                ? PlatformEvents.CHOSE_PUBLIC
-                : PlatformEvents.CHOSE_INTERNAL,
-              {
-                organization,
-                view: analyticsView,
-              }
-            );
+            if (option !== 'sentry-fx') {
+              trackIntegrationAnalytics(
```

Comment:
@vpandiarajan20 I think it makes sense to just modify this event to work with sentry functions as well.